Tasks/NodeTool/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -5,6 +5,8 @@
   "loc.instanceNameFormat": "Use Node $(versionSpec)",
   "loc.input.label.versionSpec": "Version Spec",
   "loc.input.help.versionSpec": "Version Spec of version to get.  Examples: 6.x, 4.x, 6.10.0, >=6.10.0",
+  "loc.input.label.checkLatest": "Check for latest version",
```

Comment:
Add this is typically false unless you have a specific scenario to always get latest.  This will cause it to incur download costs when potentially not necessary, especially with the hosted build pool.